OmsiHook/WrappedOmsiClasses/OmsiComplMapObjInst.cs

```diff
@@ -205,6 +213,48 @@ public void SetStringVariable(string varName, string value)
             stringVars[index] = new(value);
         }
 
+        /// <summary>
+        /// Get a constant for an object from its name.
+        /// </summary>
+        /// <param name="varName">Const Name</param>
+        /// <returns>requested float value</returns>
+        /// <exception cref="KeyNotFoundException"/>
+        public float GetConst(string varName)
+        {
+            int index = constStrings[varName];
+            if (index >= consts.Length || index < 0)
+                throw new KeyNotFoundException($"Const Variable '{varName}' not found in object. - Index Out Of Bounds");
+            return consts[index];
+        }
+
+        public float GetCurve(string varName, float x)
+        {
+            int index = funcsStrings[varName];
+            if (index >= consts.Length || index < 0)
+                throw new KeyNotFoundException($"Curve Variable '{varName}' not found in object. - Index Out Of Bounds");
+
+            for (int i = 0; i < funcs[index].Pnts.Length - 1; i++)
+            {
+                float x1 = funcs[index].Pnts[i].x;
+                float y1 = funcs[index].Pnts[i].y;
```

Comment:
Technically it would be faster to get the whole `OmsiPointPair` at once and then deconstruct it (idk the compiler might do this for you in modern versions of .net).

I wonder if this is a place where using the deconstruct pattern would make sense:
```cs
float (x, y) = pnts[i];
// In OmsiPointPair
public void Deconstruct(out float x, out float y) => //[...]
```